### DISCUSSION TITLE
Search does not return any results from code

### DISCUSSION THREAD

**Discussion by kazdy** [community/community#46906] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, 

A few weeks ago GH search stopped working for me, it does not find anything in the code but searching in issues and PRs works.

Example:
https://github.com/apache/hudi/search?q=HoodieStreamSource

Should return:
https://github.com/apache/hudi/blob/master/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieStreamSource.scala

**Comment by look** [comment, marked as answer]
@kazdy we are investigating problems with the old code search reported by users. In the meantime, I recommend you switch to the new code search beta: https://github.com/features/code-search

**Comment by pishpecki-cravit** [comment]
This error is back. On one of my private repositories. Can you tell me where can I report this?
I used Github as a very reliable tool in searching repositories. Now, I cannot be sure if the code is missing or Github search is not working.

**Reply by look** [reply]
Sorry for the inconvenience. I would first execute a search against your repositories, wait a few minutes and then try again. Our auto-indexing may fix the problem. If that doesn't fix it, you can [file a support request](https://support.github.com/request).
